mappings/net/minecraft/world/EntityView.mapping

```diff
@@ -21,30 +21,67 @@ CLASS net/minecraft/class_1924 net/minecraft/world/EntityView
 		ARG 1 entity
 		ARG 2 maxDistance
 	METHOD method_18461 getClosestPlayer (Lnet/minecraft/class_4051;DDD)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
 		ARG 2 x
 		ARG 4 y
 		ARG 6 z
 	METHOD method_18462 getClosestPlayer (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
+		ARG 2 entity
 	METHOD method_18463 getClosestPlayer (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;DDD)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
+		ARG 2 entity
 		ARG 3 x
 		ARG 5 y
 		ARG 7 z
-	METHOD method_18464 getPlayersInBox (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
+	METHOD method_18464 getPlayers (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
+		ARG 1 targetPredicate
+		ARG 2 entity
+		ARG 3 box
 	METHOD method_18466 getTargets (Ljava/lang/Class;Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
 		ARG 1 entityClass
 		ARG 2 targetPredicate
 		ARG 3 targettingEntity
 		ARG 4 box
-	METHOD method_18467 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;)Ljava/util/List;
+	METHOD method_18467 getEntitiesExcludingSpectators (Ljava/lang/Class;Lnet/minecraft/class_238;)Ljava/util/List;
 		ARG 1 entityClass
 		ARG 2 box
 	METHOD method_18468 getClosestEntity (Ljava/util/List;Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;DDD)Lnet/minecraft/class_1309;
 		ARG 1 entityList
+		ARG 2 targetPredicate
+		ARG 3 entity
 		ARG 4 x
 		ARG 6 y
 		ARG 8 z
 	METHOD method_18470 getPlayerByUuid (Ljava/util/UUID;)Lnet/minecraft/class_1657;
 		ARG 1 uuid
+	METHOD method_20743 getEntityCollisions (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/Set;)Ljava/util/stream/Stream;
+		ARG 1 entity
+		ARG 2 box
+		ARG 3 excluded
+	METHOD method_21726 getClosestEntity (Ljava/lang/Class;Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;DDDLnet/minecraft/class_238;)Lnet/minecraft/class_1309;
+		ARG 1 entityClass
+		ARG 2 targetPredicate
+		ARG 3 entity
+		ARG 4 x
+		ARG 6 y
+		ARG 8 z
+		ARG 10 box
+	METHOD method_21727 getClosestEntityIncludingUngeneratedChunks (Ljava/lang/Class;Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;DDDLnet/minecraft/class_238;)Lnet/minecraft/class_1309;
```

Comment:
including ungenerated chunks? Does this method generate those chunks and look for entities, or does it just look into the off-thread worldgen thing's entity list? Same questions below.

Reply:
This will generate chunks if necessary. The others don't generate chunks, but are used much more often (hundreds of times, vs just once) so I think they should have the shorter name.